proxy/src/identity.rs

```diff
@@ -85,6 +85,21 @@ pub fn get(api: &coco::Api, id: &coco::Urn) -> Result<Identity, error::Error> {
     Ok((api.peer_id(), user).into())
 }
 
+/// Retrieve the list of tracked identities.
+///
+/// # Errors
+pub fn list_tracked_identities(api: &coco::Api) -> Result<Vec<Identity>, error::Error> {
+    let mut tracked_users = vec![];
+    for project in api.list_projects()? {
```

Comment:
A `flat_map` might be better here, as well as a `map` on the inner loop. Then we can just `collect`. Later if we decide to de-dupe then we can just say `collect` a `HashSet` instead of a `Vec` :tada: